pkg/cloud/services/ec2/launchtemplate_test.go

```diff
@@ -309,3 +436,850 @@ func TestService_LaunchTemplateNeedsUpdate(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
In TestService_LaunchTemplateNeedsUpdate func, is it possible to add following scenarios:
- Should return true if incoming IamInstanceProfile is not same as existing IamInstanceProfile
- Should return true if incoming InstanceType is not same as existing InstanceType
- Should return false if failed to get core node security groups

Reply:
Covered 1 and 2

> Should return false if failed to get core node security groups
This piece of code is covered in other test case